config/fakro/arz-z-wave.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<Product xmlns='http://code.google.com/p/open-zwave/'>
+  <!-- Configuration -->
+  <CommandClass id="112">
```

Comment:
And from where i can get option to configuration ? In manual from roller i have no option to config.